stations.csv

```diff
@@ -70516,3 +70516,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 74727;Zwolle Stadshagen;zwolle-stadshagen;8400746;84007468;52.52764;6.05146;;NL;Europe/Amsterdam;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74727;
 74728;Boskoop;boskoop;;;52.066667;4.65;;NL;Europe/Amsterdam;t;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74728;
 74729;Huércal-Viator;huercal-viator;7156310;71563106;36.883472;-2.436222;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;56310;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74729;
+74730;Milano Famagosta;milano-famagosta;8314971;83149716;45.437730;9.168150;8483;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;8314971;f;;;;;f;;f;;f;;f;;f;;f;f;;Mailand;Milan;Milán;Milan;;;Milaan;Milán;;;ミラノ;밀라노;Mediolan;Milão;Милан;;;米兰;urn:trainline:public:nloc:csv74730;
+74731;Sorrento Circumvesuviana;sorrento-circumvesuviana;8314581;83145813;40.625544;14.379272;29034;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;8314581;t;;;;;f;;f;;f;;f;;f;;f;f;;Sorrent;;;Sorrente;;;;;;;ソレント;;;;Сорренто;;;;urn:trainline:public:nloc:csv74731;
+74732;Paestum Via Laura Mare;paestum-via-laura-mare;;;40.448580;14.973700;19697;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;8314554;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;パエストゥム;;;;Пестум;;;帕埃斯图姆;urn:trainline:public:nloc:csv74732;
+74733;Polla – Grotte di Pertosa-Auletta;polla-grotte-di-pertosa-auletta;;;40.511283;15.497249;20704;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;8314535;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;ポッラ;폴라;;;Полла;;;波拉;urn:trainline:public:nloc:csv74733;
```

Comment:
I believe the coordinates are way off here as well, https://maps.app.goo.gl/dVYVKRKRpqy8URUu5

Reply:
I have sent a message to the company that operates the coach service for further information about the exact location of this stop.